public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Todo List App created using create-react-app"
+    />
+    <title>Todo List App</title>
+  </body>
+</html>
```

Comment:
This file is missing a `<body>` and a `<div id="root"></div>`

Reply:
Done